tests/TestFramework/Coverage/CodeCoverageDataTest.php

```diff
@@ -174,6 +178,58 @@ public function test_it_throws_an_exception_when_no_coverage_found()
         $coverage->hasTests('/abc/def.php');
     }
 
+    /**
+     * @dataProvider providesFunctionSignatureLineNumbers
+     */
+    public function test_it_correctly_finds_that_line_is_signature($lineNumber)
+    {
+        $coverageData = $this->getCodeCoverageData();
+        $filePath = '/tests/Fixtures/Files/phpunit/coverage-xml/FirstLevel/firstLevel.php';
+
+        //Known function signature lines
+        $this->assertTrue(
+            $coverageData->isLineFunctionSignature($filePath,$lineNumber),
+            'isLineFunctionSignature did not return true when a line is a function signature'
+        );
+    }
+
+    public function providesFunctionSignatureLineNumbers(): array
+    {
+        return [
+            [3],
+            [19],
+            [24],
+        ];
+    }
+
+    /**
+     * @dataProvider providesNonFunctionSignatureLineNumbers
+     */
+    public function test_it_knows_when_lines_are_not_function_signature($lineNumber)
+    {
+        $coverageData = $this->getCodeCoverageData();
+        $filePath = '/tests/Fixtures/Files/phpunit/coverage-xml/FirstLevel/firstLevel.php';
+
+        //Known function signature lines
+        $this->assertFalse(
+            $coverageData->isLineFunctionSignature($filePath,$lineNumber),
```

Comment:
space after comma.
I've added `tests` folder to PHPCSFixer config and fixed all issues, could you please rebase?